farmer-credential.vm

```diff
@@ -0,0 +1,43 @@
+##Here we are injecting DateTool instance from java velocity library to set the expiry date
+#set( $defaultLocale = $date.getLocale() )
+#set( $calenderNow = $date.getCalendar() )
+#set( $ISO8601DateTimeWithMillisUTC = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'" )
+## Adding no. of Days we want to set as expiry, here '5' represent Day and '30' represent no. of days
+#set( $expiry =$calenderNow.add(5, 30))
+#set( $formattedDateOfBirth = $date.toDate("yyyy-MM-dd", ${dateOfBirth}))
+
+{
+"@context": [
+"https://www.w3.org/2018/credentials/v1",
+"https://example.com/farmer-profile-context.json"
+],
+"type": [
+"VerifiableCredential",
+"FarmerCredential"

```

Comment:
As per confluence, shouldn't it be "FarmerProfileCredential"

Reply:
updated it to farmercredential in all places, added it for consistency